cms/static/js/views/pages/container.js

```diff
@@ -241,7 +242,7 @@ function($, _, Backbone, gettext, BasePage,
         refreshPasteButton(data) {
             // Do not perform any changes on paste button since they are not
             // rendered on Library or LibraryContent pages
-            if (!this.isLibraryPage && !this.isLibraryContentPage && !self.options.isIframeEmbed) {
+            if (!this.isLibraryPage && !this.isLibraryContentPage && !this.options.isIframeEmbed) {
```

Comment:
Fixes a typo/bug introduced in https://github.com/openedx/edx-platform/pull/35587